hw/top_earlgrey/doc/design/index.md

```diff
@@ -178,6 +178,17 @@ It is also available for code storage, though that is not its intended purpose.
 
 The base address of the ROM, Flash, and SRAM are given in the address map section later in this document.
 
+### Secure boot
+
+Earlgrey follows the [Secure Boot](security/specs/secure_boot/index.md) specification. The `ROM` has 3 key slots
+which are allocated as follows:
+
+| Slot | Role | Name |
+|------|------|------|
+| 0 | test | `test 0` |
+| 1 | dev | `dev 0` |
+| 2 | prod | `prod 0` |
+
```

Comment:
@alphan @moidx What should we specify as the full keyset for Earlgrey?

My opinion: 3-2-1 for PROD/DEV/RMA keys.

Reply:
SGTM

* PROD: 3
* DEV: 2
* TEST/RMA: 1

This means we have to be very careful about how we manage the `TEST/RMA` key as there is no back up for that configuration.

Reply:
I'm not opposed to 3-2-2 (add an extra TEST/RMA key).  I think we have the space for it (both in ROM and in OTP for an enable/disable byte).  I thought I'd start with 3-2-1 and see what other people thought about it.